src/main/java/org/example/exercises/mavenExercise/Solution.java

```diff
@@ -0,0 +1,19 @@
+package org.example.exercises.mavenExercise;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Solution {
+    private static final Logger log = LogManager.getLogger(Solution.class);
+    public static void main(String[] args) {
+
+        log.info("Starting application");
+        log.info("Starting sorting data");
+        sort();
+        log.info("Closing application");
+
+    }
+    public static void sort() {
```

Comment:
where implementation of this method??? You have accepted PR from Excersies then You can merge this PR into main branch and make new branch for this changes and add loggers into existing files.